dataprofiler/data_readers/data_utils.py

```diff
@@ -564,3 +564,22 @@ def load_as_str_from_file(file_path, file_encoding, max_lines=10,
                 break
             
     return data_as_str
+
+def is_stream_buffer(filepath_or_buffer, encoding=None, seek=None):
```

Comment:
do we need encoding and seek?

Reply:
agreed, can remove those from the function input here since not being used